src/components/sidebars/info-sidebar.vue

```diff
@@ -25,6 +25,12 @@ export default {
       default: false
     }
   },
+  mounted: function() {
```

Comment:
Lets use this syntax for methods to be more in line with the rest of the codebase:

```suggestion
  mounted() {
```